packages/rlc-common/src/metadata/buildVitestConfig.ts

```diff
@@ -0,0 +1,115 @@
+import { Project } from "ts-morph";
+import { RLCModel } from "../interfaces";
+
+const nodeConfig = `export default defineConfig({
+  test: {
+    "reporters": ["basic", "junit"],
+    "outputFile": {
+      "junit": "test-results.browser.xml"
+    },
+    "fakeTimers": {
+      "toFake": ["setTimeout", "Date"]
+    },
+    "watch": false,
+    "include": ["test/**/*.spec.ts"],
+    "exclude": ["test/**/browser/*.spec.ts"],
+    "coverage": {
+      "include": ["src/**/*.ts"],
+      "exclude": [
+        "src/**/*-browser.mts",
+        "src/**/*-react-native.mts",
+        "vitest*.config.ts",
+        "samples-dev/**/*.ts"
+      ],
+      "provider": "istanbul",
+      "reporter": ["text", "json", "html"],
+      "reportsDirectory": "coverage"
+    }
+  }
+});`;
+
+const browserConfig = (options: {
+  isAzureSdkForJs: boolean;
+}) => `export default defineConfig({
+    "define": {
+      "process.env": process.env
+    },
+    ${options.isAzureSdkForJs ? `plugins: [browserMap()],` : ""}
+    "test": {
+      "reporters": ["basic", "junit"],
+      "outputFile": {
+        "junit": "test-results.browser.xml"
+      },
+      "browser": {
```

Comment:
Did you want to disable security or some other thing here across the board or just for the recorder?

Reply:
This is needed, karma did it for all browser tests